.github/workflows/ubuntu20_python.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         python -m pip install flake8 pytest
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
         pip install . -U
+    - name: Create Flag File
+      run: |
+        touch /tmp/flag
```

Comment:
I have not used this but do we really need it?

Reply:
Yes, this is needed. Might not be most efficient way to do it - but I couldn't think of a better approach yet